apis/r/R/TileDBArray.R

```diff
@@ -15,11 +15,14 @@ TileDBArray <- R6::R6Class(
     #' @param internal_use_only Character value to signal this is a 'permitted' call,
     #' as `open()` is considered internal and should not be called directly.
     #' @return The object, invisibly
-    open = function(mode=c("READ", "WRITE"), internal_use_only = NULL) {
+    open = function(mode = c("READ", "WRITE"), internal_use_only = NULL) {
       mode <- match.arg(mode)
       if (is.null(internal_use_only) || internal_use_only != "allowed_use") {
-        stop(paste("Use of the open() method is for internal use only. Consider using a",
-                   "factory method as e.g. 'SOMADataFrameOpen()'."), call. = FALSE)
+        stop(
+          "Use of the open() method is for internal use only.\n",
+          "  - Consider using a factory method as e.g. 'SOMADataFrameOpen()'.",
```

Comment:
```suggestion
          "Consider using a factory method as e.g. 'SOMADataFrameOpen()'.",
```

Reply:
This is intentional and we use this format in other places:

```
Primary message.
  - hint/suggestion 1
  - hint/suggestion 2
```